core/src/main/scala/org/apache/spark/deploy/Client.scala

```diff
@@ -221,7 +221,6 @@ object Client {
     val conf = new SparkConf()
     val driverArgs = new ClientArguments(args)
 
-    conf.set("spark.rpc.askTimeout", "10")
```

Comment:
This doesn't actually impact the default for Spark apps, but only the standalone cluster doesn't it?